reconcile/utils/ocm/base.py

```diff
@@ -16,12 +18,15 @@
 CAPABILITY_MANAGE_CLUSTER_ADMIN = "capability.cluster.manage_cluster_admin"
 
 
-class OCMModelLink(BaseModel):
+class OCMModelLinkSimple(BaseModel):
```

Comment:
suggestion: `OCMCollectionLink`
if the idea is missing, this kind of reference is for a collection of models.

Reply:
Applied in [f1a1942](https://github.com/app-sre/qontract-reconcile/pull/3775/commits/f1a1942cbf1e3a0a3b438ea3613e9e1187b62d37)